### PR TITLE
Taiga Integration

### PR DESCRIPTION
**Pull request by RicardoE105**
Improvements to #937

### CODE REVIEW

File: packages/nodes-base/nodes/Taiga/Taiga.node.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+	methods = {
+		loadOptions: {
+			// Get all the available tags to display them to user so that he can
+			// select them easily
+			async getTypes(this: ILoadOptionsFunctions): Promise<INodePropertyOptions[]> {
+				const project = this.getCurrentNodeParameter('project') as string;
+				const returnData: INodePropertyOptions[] = [];
+
+				const types = await taigaApiRequest.call(this, 'GET', `/issue-types?project=${project}`);
```
Comment (renanfilipe) [inline packages/nodes-base/nodes/Taiga/Taiga.node.ts:109]: Here I was filtering the issue types by the project id, so in order to use `projectSlug` we'll have to fetch the project id using it. To create the method `getProjectId` please see: https://taigaio.github.io/taiga-doc/dist/api.html#projects-get-by-slug
```suggestion
				const projectSlug = this.getCurrentNodeParameter('projectSlug') as string;
				const returnData: INodePropertyOptions[] = [];

				const project = await getProjectId(this, projectSlug);
				const types = await taigaApiRequest.call(this, 'GET', `/issue-types?project=${project}`);
```

File: packages/nodes-base/nodes/Taiga/issueOperationFields.ts [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+				default: '',
+				placeholder: 'product, sales',
+			},
+			{
+				displayName: 'Type',
+				name: 'type',
+				type: 'options',
+				typeOptions: {
+					loadOptionsDependsOn: [
+						'project',
```
Comment (renanfilipe) [inline packages/nodes-base/nodes/Taiga/issueOperationFields.ts:132]: Please update it to `projectSlug`.

File: packages/nodes-base/nodes/Taiga/issueOperationFields.ts [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
```
Comment (renanfilipe) [inline packages/nodes-base/nodes/Taiga/issueOperationFields.ts:261]: Please update it to `projectSlug`

**Review by renanfilipe — CHANGES_REQUESTED** [review]
Thanks for improving the nodes, I've tested the self-hosted connection and it works as expected.

File: packages/nodes-base/nodes/Taiga/issueOperationFields.ts [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+					'issue',
+				],
+				operation: [
+					'create',
+					'getAll',
+				],
+			},
+		},
+		default: '',
+		description: 'Project ID',
```
[the remark this answers is not in the document]
Comment (RicardoE105) [reply packages/nodes-base/nodes/Taiga/issueOperationFields.ts:22]: Yeah, I change the description but there is not much I can do here until we develop the proper documentation for the node and have screenshots ([example](https://docs.n8n.io/nodes/n8n-nodes-base.httpRequest/#node-reference)) for the non-technical users so they can easily find the project slug.  If you are interested in contributing with the docs for this node we have a repository for that here https://github.com/n8n-io/n8n-docs.

**Comment by RicardoE105** [comment]
@renanfilipe thanks for the feedback.

**Comment by MatheusRV** [comment]
LGFM. Could some one merge this? I need it inside N8N ASAP.

**Comment by janober** [comment]
@MatheusRV it will probably be merged the next days and then released with a new n8n version next week.

We do not simply merge code without properly testing, reviewing it and making sure that everything works consistently. If we would not, then the n8n code would become a mess and also the way the node works (as it would be totally inconsistent). I know that can be annoying as it takes longer but for us, it is much more important to release a new integration correctly than simply fast. Only like that can we make sure that n8n actually gets actually better instead of worst over time.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.83.0

**Comment by cmprmsd** [comment]
@RicardoE105 Thank you for creating this node!
I noticed it currently only supports only issues. However Taiga has many more types to offer e.g. tasks, user storys etc. 

Do you have any plans integrating these as well?
Maybe there would even be a sponsor for the effort (mentioned here: https://community.n8n.io/t/taiga-project-management-partly-created/2138)

Thanks in advance!

**Pull request merged**